src/main/core/manager.c

```diff
@@ -872,12 +872,6 @@ void manager_add_syscall_counts(Manager* manager, const Counter* syscall_counts)
     _manager_unlock(manager);
 }
 
-void manager_add_syscall_counts_global(const Counter* syscall_counts) {
```

Comment:
(no action needed) - I think this code path was because in the parallelism=1 case we used to not create any worker thread, and instead run directly from the manager thread. We tore that out though since creating a single worker thread has negligible cost and saves us from having to deal with corner cases like this one.

Reply:
Ah okay that makes sense.